lighthouse-core/gather/gatherers/accessibility.js

```diff
@@ -168,5 +175,7 @@ class Accessibility extends FRGatherer {
     });
   }
 }
-
+Accessibility.runA11yChecks = runA11yChecks;
+Accessibility.createAxeRuleResultArtifact = createAxeRuleResultArtifact;
```

Comment:
just make these static functions on Accessibility class?

Reply:
i tried out making them static fns.. but an interesting problem is that when they `.toString()` they don't have the `function ` keyword and our `.evaluate()` basically wants to treat them like fn expressions and so there's a syntax error.

instead i just exposed them via a static property.